### PR TITLE
Dcos 13228 fix ipvs

### PR DESCRIPTION
**Pull request by aeyakovenko**

### CODE REVIEW

File: src/minuteman_route_mgr.erl [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 -type state() :: state().
 -include_lib("gen_netlink/include/netlink.hrl").
--define(MINUTEMAN_TABLE, 52).
+-define(MINUTEMAN_TABLE, 255). %% local table
```
Comment (GoelDeepak) [inline src/minuteman_route_mgr.erl]: Its better to create another define for LOCAL_TABLE

File: config/sys.config [hunk 2]
```diff
@@ -22,7 +22,7 @@
   {lager, [
     {handlers, [
       {lager_console_backend, warning},
-      {lager_file_backend, [{file, "error.log"}, {level, error}]}
+      {lager_file_backend, [{file, "error.log"}, {level, debug}]}
```
Comment (GoelDeepak) [inline config/sys.config]: we would like to put it back to error

File: src/minuteman_route_mgr.erl [hunk 3]
```diff
@@ -73,9 +79,9 @@ start_link() ->
     {stop, Reason :: term()} | ignore).
 init([]) ->
     {ok, Pid} = gen_netlink_client:start_link(?NETLINK_ROUTE),
-    Routes = get_routes(Pid),
-    ensure_fib_rule(Pid),
-    {ok, #state{netlink = Pid, routes = Routes}}.
+    Iface = get_minuteman_iface(Pid),
```
Comment (GoelDeepak) [inline src/minuteman_route_mgr.erl]: Isn't interface name always "minuteman"? Why it has to be store in state?

File: src/minuteman_route_mgr.erl [hunk 4]
```diff
@@ -162,71 +168,77 @@ code_change(_OldVsn, State, _Extra) ->
...
+get_iface(Ifaces) ->
+    lager:error("Found duplicate ~p interfaces, this should never happen." ++
+                " Check your network interfaces for: ~p", [?MINUTEMAN_IFACE, [Ifaces]]),
+    {error, "Duplicate minuteman interface."}.
+
+%% see netlink.hrl for the element position
+link_msg_ifname(Msg) -> proplists:get_value(ifname, element(6, Msg)).
+link_msg_index(Msg) -> element(3, Msg).
+
+get_routes(Pid, Iface) ->
```
Comment (GoelDeepak) [inline src/minuteman_route_mgr.erl]: You could fetch all the routes for an interface. Kernel will do the filtering for you

File: src/minuteman_route_mgr.erl [hunk 5]
```diff
@@ -162,71 +168,77 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 %%% Internal functions
 %%%===================================================================
-
-get_routes(Pid) ->
+get_minuteman_iface(Pid) ->
```
Comment (GoelDeepak) [inline src/minuteman_route_mgr.erl]: There is a gen_netlink function to get interface index given the name. That will simplify you code a lot

File: src/minuteman_route_mgr.erl (hunk 3 above)
Comment (aeyakovenko) [inline src/minuteman_route_mgr.erl]: can't use the string in netlink apis, so we need to dynamically find the interface id based on the name.

File: src/minuteman_route_mgr.erl (hunk 5 above)
Comment (aeyakovenko) [inline src/minuteman_route_mgr.erl]: ah, cool!  where?

**Comment by GoelDeepak** [comment]
lgtm. Just need to rebase your commits

**Pull request merged**